.lycheeexclude-mail

```diff
@@ -0,0 +1,4 @@
+"**/*.html"
```

Comment:
`--exclude-mail` is just a flag to exclude checking mailing addresses. The `"**/*.html" "**/*.md" "**/*.txt" "**/*.json"` are the files that need to be checked.

Reply:
🤦 my bad